libs/api/domains/license-service/src/lib/licenceService.type.ts

```diff
@@ -13,6 +13,18 @@ export enum GenericLicenseType {
   Passport = 'Passport',
 }
 
+/**
+ * List of license types that can be scanned with a barcode scanner.
+ */
+export const barcodeLicenseTypes: GenericLicenseType[] = [
+  GenericLicenseType.DriversLicense,
+  GenericLicenseType.HuntingLicense,
+  GenericLicenseType.AdrLicense,
+  GenericLicenseType.MachineLicense,
+  GenericLicenseType.FirearmLicense,
+  GenericLicenseType.DisabilityLicense,
+]
```

Comment:
~The license clients already have similar logic in `clientSupportsPkPass` and `licenseIsValidForPkPass`. Would be nice to refactor to be consistent and not define this in multiple places.

Reply:
Good idea. I think I will use `clientSupportsPkPass` since that reflects the restrictions in barcodeLicenseTypes